app/assets/scss/overrides/_search-result.scss

```diff
@@ -0,0 +1,19 @@
+/**
+ * To remove our reliance on the Toolkit classes, we copy this style
+ */
+ .app-search-result {
+     padding: 0 0 15px;
+     margin: 0 0 15px;
+     border-bottom: 1px solid $govuk-border-colour;
+     @include core-16;
+     // Automatic wrapping for unbreakable text (e.g. URLs)
+     word-wrap: break-word; // Fallback for older browsers only
+     overflow-wrap: break-word;
+ }
+
+ /**
+  * To mimic the look of toolkit search results, we make metadata grey.
+  */
```

Comment:
Is this comment accurate? I thought we weren't making metadata grey any more?

Reply:
Oof, nope - from an earlier approach. Thanks for spotting!